.github/workflows/go.yml

```diff
@@ -16,6 +16,11 @@ jobs:
   generate:
     runs-on: ubuntu-latest
     steps:
+    - name: Cancel previous
+      uses: styfle/cancel-workflow-action@0.7.0
+      with:
+        access_token: ${{ github.token }}
+      if: ${{github.ref != 'refs/head/main'}}
```

Comment:
I think we can remove this line (always true), we don't have a `main` branch (yet)